docs/01_nodeos/03_plugins/http_plugin/index.md

```diff
@@ -29,6 +29,60 @@ Config Options for eosio::http_plugin:
+                                        <hostname>:port, <ipaddress>:port or
+                                        unix socket path;
+                                            in addition, unix socket path must
+                                        starts with '/', './' or '../'. When
+                                        relative path
+                                            is used, it is relative to the data
+                                        path.
+
+                                            Valid categories include chain_ro,
+                                        chain_rw, db_size, net_ro, net_rw,
+                                        producer_ro
+                                            producer_rw, snapshot, trace_api,
+                                        prometheus, and test_control.
+
+                                            A single `hostname:port`
+                                        specification can be used by multiple
+                                        categories
+                                            However, two specifications having
+                                        the same port with different hostname
+                                        strings
+                                            are always considered as
+                                        configuration error regardless of
+                                        whether they can be resolved
+                                            into the same set of IP addresses.
+
+                                          Examples:
+                                            chain_ro,127.0.0.1:8080
+                                            chain_ro,127.0.0.1:8081
+                                            chain_rw,localhost:8081 # ERROR!,
+                                        same port with different addresses
+                                            chain_rw,[::1]:8082
+                                            net_ro,localhost:8083
+                                            net_rw,server.domain.net:8084
+                                            producer_ro,/tmp/absolute_unix_path
+                                        .sock
+                                            producer_rw,./relative_unix_path.so
+                                        ck
+                                            trace_api,:8086 # listen on all
+                                        network interfaces
+
+                                          Notice that the behavior for `[::1]`
+                                        is platform dependent. For system with
+                                        IPv4 mapped IPv6 networking
+                                          is enabled, using `[::1]` will listen
+                                        on both IPv4 and IPv6; other systems
+                                        like FreeBSD, it will only
```

Comment:
I guess I never noticed this paragraph before. yuck, we should fix the discrepancy